fleece/cli/config/config.py

```diff
@@ -0,0 +1,337 @@
+            skip_import = True
+
+    if not skip_import:
+        with open(filename, 'wt') as fd:
+            export_config(args, output_file=fd)
+
+    subprocess.call(args.editor + ' ' + filename, shell=True)
+
+    with open(filename, 'rt') as fd:
+        import_config(args, input_file=fd)
+
+    os.unlink(filename)
+
+
+def render_config(args, output_file=None):
+    if not output_file:
+        output_file = sys.stdout
+    with open(args.config, 'rt') as f:
+        config = yaml.safe_load(f.read())
+    config['config'] = _decrypt_item(config['config'], stage=args.stage,
+                                     key='', render=True)
+    if args.json or args.encrypt or args.python:
+        rendered_config = json.dumps(
+            config['config'], indent=None if args.encrypt else 4,
+            separators=(',', ':') if args.encrypt else (',', ': '))
+    else:
+        buf = StringIO()
+        yaml.round_trip_dump(config['config'], buf)
+        rendered_config = buf.getvalue()
+    if args.encrypt or args.python:
+        STATE['keys'] = config['keys']
+        encrypted_config = []
+        while rendered_config:
+            buffer = _encrypt_text(rendered_config[:4096], args.stage)
+            rendered_config = rendered_config[4096:]
+            encrypted_config.append(buffer)
+
+        if not args.python:
+            rendered_config = json.dumps(encrypted_config)
+        else:
+            rendered_config = '''ENCRYPTED_CONFIG = {}
+import base64
+import boto3
+import json
+
+def load_config():
+    config_json = ''
+    kms = boto3.client('kms')
+    for buffer in ENCRYPTED_CONFIG:
+        r = kms.decrypt(CiphertextBlob=base64.b64decode(buffer.encode(
+            'utf-8')))
+        config_json += r['Plaintext'].decode('utf-8')
+    return json.loads(config_json)
+
+CONFIG = load_config()
+'''.format(encrypted_config)
+    output_file.write(rendered_config)
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(
+        prog='fleece config',
+        description=('Configuration management')
+    )
+    parser.add_argument('--config', '-c', default='config.yml',
+                        help='Config file (default is config.yml)')
+    parser.add_argument('--username', '-u', type=str,
+                        default=os.environ.get('RS_USERNAME'),
+                        help=('Rackspace username. Can also be set via '
+                              'RS_USERNAME environment variable'))
+    parser.add_argument('--apikey', '-k', type=str,
+                        default=os.environ.get('RS_API_KEY'),
+                        help=('Rackspace API key. Can also be set via '
+                              'RS_API_KEY environment variable'))
+    parser.add_argument('--environments', '-e', type=str,
+                        default='./environments.yml',
+                        help=('Path to YAML config file with defined accounts '
+                              'and stage names. Defaults to '
```

Comment:
How are stages supposed to be represented in `environments.yml`? Haven't seen any references to that in existing documentation for `fleece run`, it just lists the AWS accounts.

Reply:
I haven't made any changes to the environments file, it's exactly what the run command uses. The `name` under each environment is used as the stage name. Is that a problem?

Reply:
Thanks for the clarification. I'd prefer not mixing up the terminology. We have `environment`, `stage`, and `account`, and they seem to mean the same thing in this context. Is that really the case, or am I completely lost?

Reply:
The way I think about it is that an "environment" is an association between a stage and an AWS account.

Reply:
@szilveszter I've been thinking about this today and discussing it with Ryan. I have now added support for naming stages using regular expressions. See the updated README for an example. I'm also going to send you a screencast I created that shows how this works better.

Reply:
It looks really awesome, thank you!